ipi/inputs/outputs.py

```diff
@@ -166,6 +166,22 @@ class InputTrajectory(InputValue):
             "help": "How often should streams be flushed. 1 means each time, zero means never.",
         },
     )
+    attribs["xtratype"] = (
+        InputAttribute,
+        {
+            "dtype": str,
+            "default": "info",
+            "help": "What extra to print from the different extra strings.",
+            "options": [
+                "info",
+                "dipole",
+                "friction",
+                "hirshfeld",
+                "workfunction",
```

Comment:
maybe change it to work_function to keep it consistent with atomic_stress?

Reply:
fair point! corrected it :)